src/main/java/com/scottlogic/Main.java

```diff
@@ -20,6 +17,7 @@ public static void main(String[] args) {
     /**
      * Prints out user posts in either ascending/descending order (depending on which strategy is called)
      * @param userPosts The list of user posts to print out
+     * @param sortOrder direction in which to sort in
      */
     private static void sortUserPosts(List<UserPost> userPosts, SortOrder sortOrder) {
         SortContext sortContentLengthAscending = new SortContext(new SortContentLength());
```

Comment:
I know this is old code that I've reviewed, but the same should be applied here. Sometimes things just sneak through and reviewers catch different things.

Reply:
Fixed